### ISSUE TITLE
Docker for windows in Linux subsystem - Could not connect to the Docker daemon

### ISSUE THREAD

**Issue by cforce** [wagoodman/dive#75]
Also pure docker works, dive won't connect.

```
user@machine:/mnt/c/Users/user$ dive pgadmin4
Analyzing Image
Could not connect to the Docker daemon:unable to parse docker host `localhost:2375`
user@machine:/mnt/c/Users/user$ docker -H localhost:2375 images
REPOSITORY                                                          TAG                 IMAGE ID            CREATED             SIZE
docker4w/nsenter-dockerd                                            latest              2f1c802f322f        6 weeks ago         187kB
kdeneon/plasma                                                      user                c8b933e41707        6 weeks ago         2.59GB
kdeneon/plasma                                                      dev-unstable        1f69595e8fcb        6 weeks ago         2.59GB
openzipkin/zipkin                                                   latest              a4e6baafa76a        8 weeks ago         147MB
gcr.io/kubernetes-helm/tiller                                       v2.11.0             ac5f7ee9ae7e        2 months ago        71.8MB

```

**Comment by conradoqg** [comment]
Same thing happening here.

Here I give more context:
I configured my `Docker for windows in Linux subsystem` by setting the `DOCKER_HOST="localhost:2375"` variable and I can use docker inside the subsystem.

However, when I run dive, I receive this message:
```
conradoqg@CONRADOQG-DW:~$ dive conradoqg/cvm-fund-explorer-workers
Analyzing Image
Could not connect to the Docker daemon:unable to parse docker host `localhost:2375`
```

Best

**Comment by conradoqg** [comment]
Setting the env var to `DOCKER_HOST="tcp://localhost:2375"` solved my problem. This is connected to #84.

Cheers

**Comment by wagoodman** [comment]
This will be addressed more gracefully in #84
